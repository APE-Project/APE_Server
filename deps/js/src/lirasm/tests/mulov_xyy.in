; 46341 * 46341 >= 2^31, and will overflow.
big = immi 46341

res = mulxovi big big    ; overflow, so we exit here

; Ensure that 'big' gets its own register and isn't shared with 'res'.
; Also store 'res' so it isn't dead.
m = allocp 8
sti big m 0
sti res m 4
x                       ; we don't exit here
