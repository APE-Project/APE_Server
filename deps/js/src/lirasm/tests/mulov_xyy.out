Exited block on line: 4
